flow-typed/npm/jest_v26.x.x.js

```diff
@@ -1,5 +1,5 @@
-// flow-typed signature: e60c7806ec0ddaf4588f438843ef37bd
```

Comment:
this PR shouldn't change the flow-typed as it's only deliverying UI changes. if there is a reason to upgrade flow-typed please do in a separate PR.

Reply:
thanks! asked about this in chat , but had no response. will revert this